webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -103,6 +105,8 @@ type Props = {
     disableSend?: boolean;
 }
 
+let timeoutId: NodeJS.Timeout;
```

Comment:
Good idea to clear the timeout on unmount, but since we actually have multiple instances of `AdvanceTextEditor`, `timeoutId` should be stored in a `useRef` in the component itself

Reply:
That's true, sorry, didn't think about that. Changed it to be declared with `useRef` instead.

Reply:
No worries! This looks good to me now